modules/examiner/php/NDB_Menu_Filter_Form_examiner.class.inc

```diff
@@ -167,11 +167,12 @@ class NDB_Menu_Filter_Form_Examiner extends NDB_Menu_Filter_Form
                 // Grab their certifications from the database
                 $certifications = $DB->pselect(
                     "SELECT tn.Full_name FROM certification c
-                     LEFT JOIN test_names tn ON (tn.ID=c.testID) 
+                     RIGHT JOIN test_names tn ON (tn.ID=c.testID) 
```

Comment:
Switched to left join... did a right join since it my head the join should happen in that order, but doesn't really matter.